server/app/durablejobs/DurableJobName.java

```diff
@@ -18,7 +18,7 @@ public enum DurableJobName {
     this.jobName = jobName;
   }
 
-  public String getJobName() {
+  public String getJobNameString() {
```

Comment:
note to reviewer: renamed for clarity since otherwise we have call sites that look like `job.getJobName().getJobName()` for retrieving the string